importer/markdown/src/main/java/org/itsallcode/openfasttrace/importer/markdown/MdPattern.java

```diff
@@ -15,6 +15,8 @@ enum MdPattern
     // [impl->dsn~md.artifact-forwarding-notation~1]
 
     // @formatter:off
+    CODE_BEGIN("```\\w*\\s*"),
```

Comment:
```suggestion
    CODE_BEGIN("```\\s*"),
```
Does md allow whitespace here?

Reply:
That is from the GFM, not from standard Markdown: https://github.github.com/gfm/#fenced-code-blocks

I just realized that GFM also allows tildes.

The rules allow a lot actually. I extended the tests and the regex accordingly.